src/patterns-hmrc-govuk-team/tax-treaties/index.njk

```diff
@@ -123,6 +115,6 @@ in force</li>
 <h3 class="govuk-heading-m">Collection</h3>
```

Comment:
Can't comment on the actual line but on the above lines where we have [yyyy], you've changed that to [year] in a previous file. If you go with [year] should this file be consistent with that or is it a different thing?

Reply:
Gone with [yyyy].